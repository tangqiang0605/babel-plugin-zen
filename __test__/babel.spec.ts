import pluginTester from 'babel-plugin-tester'
import HoistCommonString from '../src/index'

pluginTester({
  plugin: HoistCommonString,
  tests: {
    'using jest snapshots': {
      code: `setTimeout(()=>{},2);setTimeout(()=>{},2);setTimeout(()=>{},2)
      
      `,
      snapshot: true,
    },
  },
})
