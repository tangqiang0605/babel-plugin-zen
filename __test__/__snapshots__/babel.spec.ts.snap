// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`zen 1. using jest snapshots: 1. using jest snapshots 1`] = `

setTimeout(()=>{},2);setTimeout(()=>{},2);setTimeout(()=>{},2)
      
      

      ↓ ↓ ↓ ↓ ↓ ↓

var _setTimeout2ms = 2
setTimeout(() => {}, _setTimeout2ms)
setTimeout(() => {}, _setTimeout2ms)
setTimeout(() => {}, _setTimeout2ms)

`;
